### ISSUE TITLE
Undeclared dependency Test::Exception

### ISSUE THREAD

**Issue by eserte** [shawnlaffan/Geo-ShapeFile#25]
The test suite of Geo-ShapeFile-2.99_001 fails if Test::Exception is not installed:
```
Can't locate Test/Exception.pm in @INC (you may need to install the Test::Exception module) (@INC contains: ...  .) at t/test_unicode_fname.t line 6.
BEGIN failed--compilation aborted at t/test_unicode_fname.t line 6.
t/test_unicode_fname.t .. 
Dubious, test returned 2 (wstat 512, 0x200)
No subtests run 
```

**Comment by shawnlaffan** [comment]
Thanks for reporting.  Should be fixed in 2.99_002, which has just been uploaded to CPAN.

**Comment by shawnlaffan** [comment]
CPAN testers is green, marking as fixed.
